sp_api/api/catalog_items/catalog_items.py

```diff
@@ -77,11 +80,14 @@ def get_catalog_item(self, asin, **kwargs) -> ApiResponse:
         Args:
             asin:string | * REQUIRED The Amazon Standard Identification Number (ASIN) of the item.
             key marketplaceIds:array | * REQUIRED A comma-delimited list of Amazon marketplace identifiers. Data sets in the response contain data only for the specified marketplaces.
-            key includedData:array |  A comma-delimited list of data sets to include in the response. Default: summaries.
+            key includedData:array |  A comma-delimited string or list of data sets to include in the response. Default: summaries.
             key locale:string |  Locale for retrieving localized summaries. Defaults to the primary locale of the marketplace.
 
         Returns:
             ApiResponse:
         """
 
+        if type(kwargs['includedData']) is list:
+            for i in kwargs['includedData']:
+                kwargs['includedData'] += f'{i},'
```

Comment:
Similarly, instead of these two lines, we can achieve the same result in one line as follows:
```python
kwargs['includedData'] = ",".join(kwargs['includedData'])
```
It's also more Pythonic this way.